package.json

```diff
@@ -13,19 +13,15 @@
     "api:wsl": "nodemon --legacy-watch api/bin/www",
     "api:build": "cd api && tsc",
     "api:watch": "cd api && tsc --watch",
-
```

Comment:
@crhallberg put this whitespace here for clarity, but I anticipate that it's going to get auto-reverted and accidentally committed endlessly. @crhallberg, should we just bite the bullet and let it get flattened out so we don't spend time constantly fixing it? Or do you think it's worth the effort?

Reply:
Yeah we don't need to fight it